servers.json

```diff
@@ -366,6 +366,13 @@
             "shotbow.net"
         ]
     },
+    {
+        "id": "crystal",
+        "name": "Crystal Network",
+        "addresses": [
+            "crystal.rip",
```

Comment:
Skid can't comprehend how JSON works and adds a comma there 😂😂😂

Reply:
Simply just didn’t get added in the json by accident